static/usage/v7/content/theming/safe-area/angular/global_css.md

```diff
@@ -0,0 +1,19 @@
+```css
+:root {
+  /**
+   * Setting the variables for DEMO purposes only.
+   * Values will be set automatically when building an iOS or Android app.
+   */
+  --ion-safe-area-top: 20px;
+  --ion-safe-area-bottom: 20px;
+  --ion-safe-area-left: 20px;
+  --ion-safe-area-right: 20px;
+}
+
+ion-content::part(scroll) {
+  padding-top: var(--ion-safe-area-top, 0);
+  padding-bottom: var(--ion-safe-area-bottom, 0);
+  padding-left: var(--ion-safe-area-left, 0);
+  padding-right: var(--ion-safe-area-right, 0);
+}
```

Comment:
I think we should keep this CSS in the example component's CSS file. Developers often have an app where one page has a content with no header but the rest do, so I don't think we want this globally applied.

Reply:
Then i suppose we should also have two `<style>` tags in the Vue file, one that's scoped and one that's not

Reply:
I'll separate them on Vue. It keeps it consistent IMO.